README.md

```diff
@@ -711,7 +711,8 @@ but you want the logs to be readable by specific users.
 When you want to make sure that user has some software installed or has the
 right version you can write a simple script and put it into checks directory.
 All files found there will be ran and if any has non-zero exit code, kafo
-wont execute puppet.
```

Comment:
I'm not native English speaker, but i think a slightly better wording would be:

All files found there will be executed and if any of these exits with an non-zero exit code, kafo won't execute puppet but print an error message `Your system does not meet configuration criteria`.